src/python/pants/backend/python/macros/python_artifact.py

```diff
@@ -87,11 +74,6 @@ def __init__(self, **kwargs):
         # Instead we stringify and precompute a hash to use in our own __hash__, since we know
         # that this object is immutable.
         self._hash: int = hash(json.dumps(kwargs, sort_keys=True))
-        self._name: str = name
-
-    @property
-    def name(self) -> str:
```

Comment:
I ran all the tests, and manually built some dists, to verify that this property isn't used.